### DISCUSSION TITLE
Curl objects to create Ravendb database with multiple json objects

### DISCUSSION THREAD

**Discussion by Kannadasa** [ravendb/ravendb#14659] [General]
I am using CURL command to create a database using Json object.  If i mention only database name in the curl command it works fine and creates a database with 1 Replication factor.  If i have to create a database with Replication factor =3 then CURL command is not creating with replication factor=3, it always defaults to 1.

 curl.exe -X PUT -H "Content-Type: application/json" -d '{\"DatabaseName\": \"DB_Kanna3\", \"ReplicationFactor\": \"3\"}' --cert "LocalMachine\My\xxxxxx" --key "D:\RavenDB\certificates\xxx.key" https://xxx:000/admin/databases

By this command it is creating database with replication factor =1 and adding to only one node.

I need the proper syntax to create a database with replication factor =3

Thanks
Kannadasan

**Comment by Kannadasa** [comment]
Do we know the Key value pair names we need to define within json body for replication factor in the curl command?

**Reply by ayende** [reply]
https://github.com/ravendb/ravendb/blob/v5.4/src/Raven.Client/ServerWide/Operations/CreateDatabaseOperation.cs#L65
